src/main/ml-modules/ext/serviceModel.sjs

```diff
@@ -1008,7 +1008,8 @@ function _buildOneLayerDescriptor(serviceName, layerModel) {
     if (DEBUG) xdmp.trace("GDS-DEBUG", Sequence.from(["layerModel:", layerModel.id]));
     const layer = {
       metadata: {
-        maxRecordCount: MAX_RECORD_COUNT
+        maxRecordCount: MAX_RECORD_COUNT,
```

Comment:
I think what would help a lot is a link to the spec we're adhering to - i.e. how do we know that "metadata" with children of "maxRecordCount" and "useStandardizedQueries" is valid? Check with @jkerr5  on that. I think a link to that in this function would be useful here at a minimum.